launcher/verifier/rest/rest.go

```diff
@@ -83,7 +83,7 @@ func (c *restClient) CreateChallenge(_ context.Context) (*verifier.Challenge, er
 		&v1alpha1.Challenge{},
 	).Do()
 	if err != nil {
-		return nil, fmt.Errorf("calling v1alpha1.CreateChallenge: %w", err)
+		return nil, handleError(err, "Error calling v1alpha1.CreateChallenge")
```

Comment:
these error messages will get further wrapped (in https://github.com/google/go-tpm-tools/blob/e0f2e6b3305071ba6e6a006e9687b127092f8ec1/launcher/container_runner.go#L344). They should not be capitalized: https://google.github.io/styleguide/go/decisions#error-strings